mono/metadata/console-unix.c

```diff
@@ -254,7 +254,8 @@ do_console_cancel_event (void)
 	method = mono_class_get_method_from_name (klass, "BeginInvoke", -1);
 	g_assert (method != NULL);
 
-	mono_threadpool_ms_begin_invoke (domain, (MonoObject*) load_value, method, NULL);
+	mono_threadpool_ms_begin_invoke (domain, (MonoObject*) load_value, method, NULL, &error);
+	mono_error_cleanup (&error);
```

Comment:
Well, what's the expected behavior.
Maybe an exception/error on that case should abort the runtime.